Python/Product/ExternalProfilerDriver/VTuneStackParser.cs

```diff
@@ -34,7 +37,12 @@ class PerformanceSample {
 
         public PerformanceSample(string function, string cpuTime, string module, string functionFull, string sourceFile, string startAddress) {
             Function = function;
-            CPUTime = Single.Parse(cpuTime);
+            try {
+                CPUTime = Single.Parse(cpuTime);
+            } catch (Exception) { // TODO -- should use a more specific exception
```

Comment:
check https://docs.microsoft.com/en-us/dotnet/api/system.single.parse?view=netframework-4.7.2#System_Single_Parse_System_String_System_IFormatProvider_ for the exceptions to catch

You should also specify an IFormatProvider. If this comes from a file, then I hope this is culture invariant? If that's the case, then pass `CultureInfo.InvariantCulture`